crates/autopilot/src/run_loop.rs

```diff
@@ -638,6 +644,11 @@ struct Metrics {
     /// Tracks the result of driver `/settle` requests.
     #[metric(labels("driver", "result"))]
     settle: prometheus::IntCounterVec,
+
+    /// Tracks the number of orders that were part of some but not the winning
+    /// solution.
+    #[metric(labels("winner"))]
```

Comment:
This label is confusing to me.
The grafana metric would show a list of orders associated with a given driver which did **NOT** settle these orders. Is this intended?
If this is the case could the label have a easier to understand name? Maybe `ignored_by` or sth like that?
Just seems to me that you would have to have a lot of context to understand the metric only looking at grafana.
Also adding more doc comments on the metric might not help since they get cut off at a certain length IIRC.